src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,65 @@
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { useCallback, useEffect, useState } from 'react';
+
+export const PeoplePage = () => {
+  const [error, setError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const fetchPeopleFromApi = useCallback(async () => {
+    try {
+      const todosFromApi = await getPeople();
+
+      setPeople(todosFromApi);
+    } catch {
+      setError(true);
```

Comment:
Consider logging the error or providing more detailed error information to help with debugging. Currently, the error handling only sets a boolean state, which might not be sufficient for diagnosing issues.